ts/private/ts_project.bzl

```diff
@@ -29,6 +29,15 @@ def _gather_types_from_js_infos(targets):
     ])
     return depset([], transitive = files_depsets)
 
+def _gather_transitive_typecheck_from_output_group_infos(typecheck_outs, targets):
+    files_depsets = []
+    files_depsets.extend([
```

Comment:
I was copy-pasting the code above but I guess I can just assign the list directly